### DISCUSSION TITLE
Make `pixi run uv` globally available

### DISCUSSION THREAD

**Discussion by liquidcarbon** [prefix-dev/pixi#992] [Ideas]
Since you've switched to `uv` in 0.16 and a lot of people are becoming familiar with `uv`, would it be worth making it available in the CLI?
- `pixi run uv`
- `pixi uv`
- `uv`
- etc.?

**Comment by wolfv** [comment]
You can already do `pixi global install uv` and then run it from the command line :)

**Comment by liquidcarbon** [comment]
So I've been practicing pixi + uv workflows in Pixi projects. #968 
In the absence of `pip freeze` equivalent #518 I'm exploring various hacks.

**pixi project with global uv**
- `uv pip freeze` and `uv pip compile` look for python interpreter inside `~/.pixi/envs/uv/bin` and of course not find it

**pixi project with LOCAL uv (`pixi add uv`)**
- `uv` still calls the global `uv`
- `pixi run uv` works as expected

So... `pixi run uv` as an alternative installer to `pixi add` seems quite useful.

Because pixi packages uv, why don't you make `pixi run uv` or `pixi uv` always available, without a separate installation of `uv` (global or local)?

**Reply by baszalmstra** [reply]
Im not quite sure what you are trying to accomplish here. `pixi add` and `pixi run uv` are not equivalent commands. 

If you are trying to add a wheel directly from your filesystem or from an URL into your environment, that is currently supported by `pixi` on the `main` branch. Will be released pretty soon.

**Reply by liquidcarbon** [reply]
Sorry, maybe I shouldn't have renamed the topic, made it more confusing.

I guess I'm just wondering if it's possible to get `uv` with `pixi` automatically, without specifically installing `uv` - given that `pixi` uses it in the background.

**Reply by baszalmstra** [reply]
Pixi does not actually bundle uv. It does not invoke uv, pixi uses the same internals as uv but doesnt actually use the uv cli at all.

**Reply by liquidcarbon** [reply]
Got it, now I'm clear on that.
